.github/workflows/e2e.yaml

```diff
@@ -37,7 +37,11 @@ jobs:
         KO_DOCKER_REPO="" go run ./ build --push=false ./test | grep ":latest@sha256:"
         KO_DOCKER_REPO="" go run ./ build --push=false -t test ./test | grep ":test@sha256:"
         KO_DOCKER_REPO="" go run ./ build --push=false -t test --tag-only ./test | grep ":test$"
-
+        
+        # Check that using sbom-dir works.
+        KO_DOCKER_REPO=""  go run ./ build -t test --push=false --sbom-dir ./sbom-data ./test
+        jq . ./sbom-data/*.json
```

Comment:
I think now we could test that this is `./sbom-data/test.json` right?

Otherwise LGTM!